build.gradle

```diff
@@ -37,12 +36,18 @@ apply plugin: 'kotlin-android'
 android {
     namespace 'com.mparticle.kits.apptentive'
     defaultConfig {
-        minSdkVersion 16
```

Comment:
Why changing the minSdk version? For every other sdk our minSdk is 16.

Reply:
The refactor of this mP SDK is to include new Apptentive android SDK 6.x and that comes with the system requirements of `minSDKVersion 21` https://learn.apptentive.com/knowledge-base/android-quick-start-guide/#system-requirements